templates/registry/markdown/resource_macros.j2

```diff
@@ -0,0 +1,9 @@
+{#- Macros for simplifying creating "Resource" documentation. -#}
+{% import 'stability.j2' as stability %}
+{% macro header(resource) %}
+**Status:** {{ stability.badge(resource.stability, resource.deprecated) }}
+
+**type:** `{{ resource.name }}`
```

Comment:
Maybe we should use `display_name` and fall back to `name` if the display name doesn’t exist. I'm not sure.

This also leads me to think that sometimes we have a `display_name` without the definition of `name` in the semconv (e.g. `registry/container.yaml`). Perhaps we should consider adding this kind of rule in the future.

Reply:
I use `name` because it must be unique here.  Type, eventually, will be used formally in merge rules, see: https://github.com/open-telemetry/oteps/pull/264